### ISSUE TITLE
Update Greek translations

### ISSUE THREAD

**Issue by geopolitis** [squidfunk/mkdocs-material#5523]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "el",
  "action.edit": "Επεξεργασία αυτής της σελίδας",
  "action.skip": "Μετάβαση στο περιεχόμενο",
  "action.view": "Προβολή πηγαίου κώδικα",
  "announce.dismiss": "Μην το ξαναδείξετε αυτό",
  "blog.archive": "Aρχείο",
  "blog.categories": "Κατηγορίες",
  "blog.categories.in": "Στο",
  "blog.continue": "Περισσότερα",
  "blog.draft": "Πρόχειρο",
  "blog.index": "Eπιστροφή",
  "blog.meta": "Metadata ⬅️",
  "blog.references": "Σχετικοί σύνδεσμοι",
  "clipboard.copy": "Αντιγραφή στο πρόχειρο",
  "clipboard.copied": "Αντιγράφηκε στο πρόχειρο",
  "consent.accept": "Αποδοχή",
  "consent.manage": "Περισσότερες επιλογές",
  "consent.reject": "Απόρριψη",
  "footer": "Υποσέλιδο",
  "footer.next": "Επόμενο",
  "footer.previous": "Προηγούμενο",
  "header": "Κεφαλίδα",
  "meta.comments": "Σχόλια",
  "meta.source": "Πηγή",
  "nav": "Πλοήγηση",
  "readtime.one": "1 λεπτό διάβασμα",
  "readtime.other": "# λεπτά διάβασμα",
  "rss.created": "Ροές Δεδομένων RSS",
  "rss.updated": "Ροές Δεδομένων RSS. Τελευταία νέα",
  "search": "Αναζήτηση",
  "search.placeholder": "Αναζήτηση",
  "search.share": "Διαμοίραση",
  "search.reset": "Καθαρισμός",
  "search.result.initializer": "Αρχικοποίηση αναζήτησης",
  "search.result.placeholder": "Πληκτρολογήστε για να αρχίσει η αναζήτηση",
  "search.result.none": "δεν βρήκε κάποιο έγγραφο",
  "search.result.one": "1 έγγραφο που ταιριάζει",
  "search.result.other": "# έγγραφα που ταιριάζουν",
  "search.result.more.one": "1 ακόμα σε αυτήν τη σελίδα",
  "search.result.more.other": "# ακόμα σε αυτήν τη σελίδα",
  "search.result.term.missing": "Λείπει",
  "select.language": "Επιλογή γλώσσας",
  "select.version": "Επιλογή έκδοσης",
  "source": "Μετάβαση στο αποθετήριο",
  "source.file.contributors": "Συνεισφέροντες",
  "source.file.date.created": "Δημιουργήθηκε",
  "source.file.date.updated": "τελευταία ενημέρωση",
  "tabs": "Καρτέλες",
  "toc": "Πίνακας περιεχομένων",
  "top": "Επιστροφή στην αρχή"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by geopolitis** [comment]
I translated some of the missing words/messages.

**Comment by squidfunk** [comment]
Perfect, thanks! Added in d10badb0b.

**Comment by squidfunk** [comment]
Released as part of [9.1.14](https://github.com/squidfunk/mkdocs-material/releases/tag/9.1.14).
